packages/editor/src/bindings/post-meta.js

```diff
@@ -12,33 +12,17 @@ import { store as editorStore } from '../store';
 export default {
 	name: 'core/post-meta',
 	label: _x( 'Post Meta', 'block bindings source' ),
-	useSource( props, sourceAttributes ) {
-		const { getCurrentPostType } = useSelect( editorStore );
-		const { context } = props;
-		const { key: metaKey } = sourceAttributes;
+	getPlaceholder( { args } ) {
+		return args.key;
+	},
+	getValue( { registry, context, args } ) {
```

Comment:
In the backend, we have `get_value_callback` which receives "args" and the "block instance". while registry is fine and is a client only thing, I wonder if we should use the same argument between client and server.